torcharrow/functional.py

```diff
@@ -4,12 +4,14 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+from msilib.schema import Error
```

Comment:
This looks not needed? (And cause test failure )

Reply:
Yup just removed!